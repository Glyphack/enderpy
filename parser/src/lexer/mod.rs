mod diagnostics;
pub mod fstring;
pub mod lexer;
