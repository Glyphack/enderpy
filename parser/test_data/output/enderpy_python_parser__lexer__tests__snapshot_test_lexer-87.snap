---
source: parser/src/lexer/mod.rs
description: break class continue def del elif else except
---
0,5: Break                                (None)
6,11: Class                               (None)
12,20: Continue                        (None)
21,24: Def                                  (None)
25,28: Del                                  (None)
29,33: Elif                                (None)
34,38: Else                                (None)
39,45: Except                            (None)
